deploy/kubernetes/base/node-daemonset.yaml

```diff
@@ -1,32 +1,36 @@
 ---
-
-kind: DaemonSet
+# Source: aws-fsx-csi-driver/templates/node-daemonset.yaml
 apiVersion: apps/v1
+kind: DaemonSet
 metadata:
   name: fsx-csi-node
-  namespace: kube-system
+  labels:
+    app.kubernetes.io/name: aws-fsx-csi-driver
 spec:
   selector:
     matchLabels:
       app: fsx-csi-node
+      app.kubernetes.io/name: aws-fsx-csi-driver
```

Comment:
I don't know if this is a concern but this will need intervention on a new apply since matchLabels are immutable.